tests/unit/test_s3.py

```diff
@@ -116,72 +116,6 @@ def test_expand_multipart_filename(self):
         self.assertIn(b'uploads/20170826T181315.679087009Z/upload/pixel.txt', expanded3,
             'Should see the interpolated filename')
 
-    def test_get_bucket_name(self):
```

Comment:
Can we try to leave this test in the code base - I think this is indeed a useful test, which can help us avoid running into regressions. (We can adapt the logic to make the tests pass, if needed.) Wdyt?

Reply:
Moved the test case to new function with updated regex.